program-management-tracker.md

```diff
@@ -1,6 +1,6 @@
 # Operate First community plan status
```

Comment:
Looks like this commit sneaked in from PR #110 
Once the other PR is merged, it will disappear from the GH UI. Or, you rebase in another branch